benchmarks/scripts/run_local_benchmarks.sh

```
@@ -1,42 +1,314 @@
 #!/usr/bin/env zsh
```

Shellcheck (0.11.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)